gitpod-io/workspace-images
# refactor(lang-rust): Simplify lang-rust setup

## Description

Instead of wrapping `rustup` in a shell script(was done in [#808](https://github.com/gitpod-io/workspace-images/pull/808/files#diff-a110d763a3b6360e4d92cc4ed09232d4e13826c6e99d564b9ce95e3f58a23a82) and caused errors after first run because I forgot to put shebang on recreation of the script), we can just move it into `/workspace/.cargo/bin` on workspace startup, which also works fine it seems. https://open-vsx.org/extension/rust-lang/rust does not like `rustup` being a shell script. 

## Related Issue(s)

Fixes [#Couldn't start rust language server](https://discord.com/channels/816244985187008514/816246578594840586/974169601824935936)
https://github.com/gitpod-io/workspace-images/issues/832
#841

## How to test

- Open https://github.com/axonasif/test/tree/rustimg-test on Gitpod
- Check if `Rust` and `rust-analyzer` VSCODE extensions are working fine (It's not ideal to install both these extensions together but doing here for testing. `matklad.rust-analyzer` will eventually be replaced with `rust-lang.rust` officially but `rust-lang.rust` is still being used by some people, so we need to test that too for now)

## Release Notes

```release-note
```

## Documentation
